dts/bindings/timer/andestech,atcpit100.yaml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2022, Andes Technology Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description:
+    This is a representation of the Andes Technology atcpit100 PIT nodes
```

Comment:
This is not an allowed style for description strings. Please read https://docs.zephyrproject.org/latest/build/dts/bindings.html#general-rules.

Reply:
Fixed it, thank you.